docs/guide-zh-CN/input-forms.md

```diff
@@ -233,8 +233,8 @@ Pjax::end();
 
 #### 提交按钮和文件上传中的值
 
-在处理 [[https://github.com/jquery/jquery/issues/2321|files]] 和
-[[https://github.com/jquery/jquery/issues/2321|submit button values]] 
+在处理 (files)[https://github.com/jquery/jquery/issues/2321] 和
+(submit button values)[https://github.com/jquery/jquery/issues/2321] 
```

Comment:
The brackets are in wrong order, link title should be in `[]` and URL in `()`. A good way to remember this is that `[]` are valid characters in a URL, while `()` are not, so URL must be wrapped in `()`.

Reply:
I'm sorry, I will fix it.